bindings/puzzles.json

```diff
@@ -185,5 +202,51 @@
       "inner_puzzle_hash": "Bytes32"
     },
     "return": "Bytes32"
+  },
+  "StreamedCat": {
+    "type": "class",
+    "new": true,
+    "fields": {
+      "coin": "Coin",
+      "asset_id": "Bytes32",
+      "proof": "LineageProof",
+      "inner_puzzle_hash": "Bytes32",
+      "recipient": "Bytes32",
+      "clawback_ph": "Option<Bytes32>",
+      "end_time": "u64",
+      "last_payment_time": "u64"
+    },
+    "methods": {
+      "amount_to_be_paid": {
+        "args": {
+          "payment_time": "u64"
+        },
+        "return": "u64"
+      },
+      "spend": {
```

Comment:
This should be a method on `Clvm` - which wraps `SpendContext` and you can insert the coin spend directly (see `spend_nft` as an example).